wannierberri/__tabulate.py

```diff
@@ -358,6 +358,9 @@ def plot_path_fat(self,
                     selE=(e<=Emax)*(e>=Emin)
                     klineselE=kline[selE]
                     y=data[selE][:,ib]
+                    select = np.abs(y) > 2
+                    y[select] = np.log2(y[select])
+                    y[~select] *= 0.5
```

Comment:
Please add a comment (in the docstring for `plot_path_fat`) that the circle size changes  linearly below 2 and logarithmically above 2.

Reply:
Done